.github/workflows/ci.yml

```diff
@@ -32,14 +32,50 @@ jobs:
   build:
     name: Execute CI script
     runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        target:
+          - armv7-unknown-linux-gnueabi
+          - armv7-unknown-linux-gnueabihf
+          - arm-unknown-linux-gnueabi
+          - aarch64-unknown-linux-gnu
+          - i686-unknown-linux-gnu
+          - loongarch64-unknown-linux-gnu
+          - powerpc64-unknown-linux-gnu
+          - powerpc64le-unknown-linux-gnu
+          - x86_64-pc-windows-msvc
+          - x86_64-apple-darwin
+          - aarch64-apple-darwin
+          - x86_64-unknown-freebsd
+          - riscv64gc-unknown-linux-gnu
```

Comment:
we are missing the "default" target here now. Something like `x86_64-unknown-linux-gnu`, which was probably excluded from the previous list as it was running when there was no `--target` provided (and which now builds when we call `cargo test`.